presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlIntegrationSmokeTest.java

```diff
@@ -288,6 +288,34 @@ public void testInsertIntoNotNullColumn()
         assertUpdate("DROP TABLE test_insert_not_null");
     }
 
+    @Test
+    public void testAggregationPushdown()
```

Comment:
Also we need some smoke tests for aggregation push down added to base jdbc smoke test that could be reused by other connectors. Such test should handle aggregration rewrites that are implemented in base-jdbc module.

Reply:
There are no rewrites handlded on base-jdbc level. There are rewrite rules provided by base-jdbc, but it's up to specific connector to mix & match them (and test them).